curriculum/challenges/_meta/basic-html-and-html5/meta.json

```diff
@@ -100,6 +100,10 @@
       "bad87fee1348bd9aedf08835",
       "Create a Set of Checkboxes"
     ],
+    [
+      "5c6c06847491271903d37cfd",
```

Comment:
Just want to make sure, you've create the challenge key using the Mongo tools in the command line. Right

Reply:
Yes, I ran ObjectId() from the command line as [shown here](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-curriculum-challenges.md#numbering-challenges).

Reply:
Great!